src/compo/viirs_aod2ioda.py

```diff
@@ -13,10 +13,57 @@
 import numpy as np
 import os
 
+import glob
```

Comment:
this isn't used?

Reply:
Thanks for catching that import, it was left over from an earlier way of creating the list of ADP files. I have removed it in the newest commit.